test/fuse/README.md

```diff
@@ -101,3 +148,41 @@ SyscallFailsWithErrno(...)
 
 Please refer to [test/syscalls/README.md](../syscalls/README.md) for further
 details.
+
+## Writing a new FuseTestCmd
```

Comment:
Acronyms like FUSE should be in all caps or all lower case in code:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Please make this change everywhere.

Reply:
[super nit]: `FuseTestCmd` is a bit confusing. maybe `FUSEInspect` is better? Naming is hard

Reply:
> Acronyms like FUSE should be in all caps or all lower case in code:
> https://github.com/golang/go/wiki/CodeReviewComments#initialisms
> 
> Please make this change everywhere.

This is a cc file and should refer to [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules).
> For the purposes of the naming rules below, a "word" is anything that you would write in English without internal spaces. This includes abbreviations, such as acronyms and initialisms. For names written in mixed case (also sometimes referred to as "camel case" or "Pascal case"), in which the first letter of each word is capitalized, prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC().